.deployment/files/realms.yaml

```diff
@@ -5,6 +5,12 @@
 
 path: ''
 name: 'Welcome to Tobira'
+page_admins:
+  - ROLE_USER_SABINE
+  - ROLE_USER_BJORK
+page_moderators:
+  - ROLE_USER_MORGAN
+  - ROLE_USER_JOSE
```

Comment:
Can you make the roles in our test data a bit more interesting? Like: I'm fine with only admin able to modify the start page, for example. Then, for different children of the root realm, you can give various users various abilities.  Also with adding more abilities for the grandchildren and so on. So basically that we can already test these inheritance things properly with test data. And maybe give access to a group (like ROLE_STAFF) to one realm (maybe not one with super many children).